tests/cupy_tests/creation_tests/test_basic.py

```diff
@@ -35,6 +41,12 @@ def test_empty_int(self, xp, dtype, order):
         a.fill(0)
         return a
 
+    @testing.slow
+    def test_empty_huge_int(self):
+        a = cupy.empty(2 ** 31, dtype='b')
+        a.fill(0)
+        self.assertFalse(a.any())
+
```

Comment:
1. OK, it makes sense.
2. Thank you for the fix.
But I think `fill(0)` should still be tested as well, because the internal implementation is separated from `fill(n)` as I said.

How about the meaning of `_int` suffix?
My guess is, you meant "huge int" which is used as an index, and did not intend to align with `_int` suffixes in other test cases. If so, it could cause a confusion and I think it should be renamed as `_huge_size` or alike.